RipCurrent/test.cc

```diff
@@ -298,8 +298,8 @@ TEST(RipCurrent, TypeSystemGuarantees) {
   using namespace ripcurrent_unittest;
 
   EXPECT_EQ("Foo", Foo::UnitTestClassName());
-  EXPECT_TRUE(Foo::INPUT_POLICY == InputPolicy::DoesNotAccept);
-  EXPECT_TRUE(Foo::OUTPUT_POLICY == OutputPolicy::Emits);
+  static_assert(Foo::INPUT_POLICY == InputPolicy::DoesNotAccept, "");
```

Comment:
The `TypeList` part of the `OUTPUT` has to be a `static_assert`, and I just changed the `INPUT` to look the same way.

Overall, if it can fail at compile time, why not let it? :-)